src/components/RouteUpdate.tsx

```diff
@@ -0,0 +1,320 @@
+        peerID = p ? p.ip : ''
+
+        return {
+            id: formRoute.id,
+            network: formRoute.network,
+            network_id: formRoute.network_id,
+            description: formRoute.description,
+            peer: peerID,
+            enabled: formRoute.enabled,
+            masquerade: formRoute.masquerade,
+            metric: formRoute.metric
+        } as Route
+    }
+
+    const handleFormSubmit = () => {
+        form.validateFields()
+            .then((values) => {
+                const routeToSave = createRouteToSave()
+                dispatch(routeActions.saveRoute.request({getAccessTokenSilently:accessToken, payload: routeToSave}))
+            })
+            .catch((errorInfo) => {
+                console.log('errorInfo', errorInfo)
+            });
+    };
+
+    const setVisibleNewRoute = (status:boolean) => {
+        dispatch(routeActions.setSetupNewRouteVisible(status));
+    }
+
+    const onCancel = () => {
+        if (savedRoute.loading) return
+        setEditName(false)
+        dispatch(routeActions.setRoute({
+            network: '',
+            network_id: '',
+            description: '',
+            peer: "",
+            metric: 9999,
+            masquerade: false,
+            enabled: false
+        } as Route))
+        setVisibleNewRoute(false)
+    }
+
+    const onChange = (data:any) => {
+        setFormRoute({...formRoute, ...data})
+    }
+
+    const handlePeerChange = (value: string) => {
+            setFormRoute({
+                ...formRoute,
+                peer: value
+            })
+    };
+
+
+    const handleChangeEnabled = ({ target: { value } }: RadioChangeEvent) => {
+        setFormRoute({
+            ...formRoute,
+            enabled: value
+        })
+    };
+
+    const handleChangeMasquerade = ({ target: { value } }: RadioChangeEvent) => {
+        setFormRoute({
+            ...formRoute,
+            masquerade: value
+        })
+    };
+
+
+    const dropDownRender = (menu: React.ReactElement) => (
+        <>
+            {menu}
+        </>
+    )
+
+    const toggleEditName = (status:boolean) => {
+        setEditName(status);
+    }
+
+    const toggleEditDescription = (status:boolean) => {
+        setEditDescription(status);
+    }
+
+    return (
+        <>
+            {route &&
+                <Drawer
+                    headerStyle={{display: "none"}}
+                    forceRender={true}
+                    visible={setupNewRouteVisible}
+                    bodyStyle={{paddingBottom: 80}}
+                    onClose={onCancel}
+                    autoFocus={true}
+                    footer={
+                        <Space style={{display: 'flex', justifyContent: 'end'}}>
+                            <Button onClick={onCancel} disabled={savedRoute.loading}>Cancel</Button>
+                            <Button type="primary" disabled={savedRoute.loading} onClick={handleFormSubmit}>{`${formRoute.id ? 'Save' : 'Create'}`}</Button>
```

Comment:
When creating a new route it is not added to the table, it is just empty.
While the APi returns the list of routes

Reply:
did you create a rule with peer in it?

Reply:
Both with and without

Reply:
checked here and in all cases, it worked fine. Please pull the latest changes from the remote repo